spec/std/hash_spec.cr

```diff
@@ -88,6 +88,46 @@ describe "Hash" do
     end
   end
 
+  describe "<" do
+    it do
+      h1 = {"a" => 1, "b" => 2}
+      h2 = {"a" => 1, "b" => 2, "c" => 3}
+      (h1 < h2).should be_true
+      (h2 < h1).should be_false
+      (h1 < h1).should be_false
```

Comment:
Tests for the empty case are needed. 

Is there any need of tests for the case when `a > b` and `b > a` are more different in both ways? ie for example `a = {"a" => 1}; b = {"b" => 2}`?

Reply:
@ysbaddaden Is there a reason you marked this conversation as _resolved_?